sources/packages/web/src/views/aest/student/applicationDetails/ApplicationRestrictionsManagement.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Student Applications"
+        :routeLocation="{
+          name: AESTRoutesConst.STUDENT_APPLICATIONS,
+          params: { studentId },
+        }"
+        subTitle="Application Restriction Management"
+      />
+      <div v-if="applicationDetail.applicationNumber">
+        <span class="label-bold-normal">Application #: </span>
+        <span
+          >{{ emptyStringFiller(applicationDetail.applicationNumber) }} |
```

Comment:
Do we need the `emptyStringFiller` here if you are checking for the `applicationNumber` in `v-if`.

Reply:
This component is replaced with <application-header-title :application-id="applicationId" />.